src/main/resources/org/sagebionetworks/web/client/widget/RadioWidgetViewImpl.ui.xml

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE ui:UiBinder SYSTEM "http://dl.google.com/gwt/DTD/xhtml.ent">
+<ui:UiBinder xmlns:ui="urn:ui:com.google.gwt.uibinder"
+	xmlns:g="urn:import:com.google.gwt.user.client.ui" 
+	xmlns:b="urn:import:org.gwtbootstrap3.client.ui"
+	xmlns:bh="urn:import:org.gwtbootstrap3.client.ui.html"
+	xmlns:t="urn:import:org.sagebionetworks.web.client.view.bootstrap.table">
+	<ui:with field='sageImageBundle' type='org.sagebionetworks.web.client.SageImageBundle' />
+	<t:Table width="100%">
+		<t:TableRow>
+			<t:TableData width="30px" addStyleNames="center">
+				<b:Radio name="radio" ui:field="radio"/>
```

Comment:
multiple radio groups showing at the same time.  should be able to set radio group name per set.